pkg/ts/Runtime.ts

```diff
@@ -8,15 +8,15 @@
 import {Arc} from './core/Arc.js';
 import {Host} from './core/Host.js';
 import {Store} from './core/Store.js';
+import {ArcMeta, ParticleMeta, StoreMeta} from './core/Meta.js';
 import {EventEmitter} from './core/EventEmitter.js';
-import {Paths} from './utils/paths.js';
-import {AggregateStore} from './core/AggregateStore.js';
-import {logFactory} from './utils/log.js';
+import {logFactory, Logger} from './utils/log.js';
 import {makeId} from './utils/id.js';
+import {Dictionary} from './utils/object.js';
 
 const log = logFactory(logFactory.flags.runtime, 'runtime', 'forestgreen');
 
-type Dictionary<T> = {[name: string]: T};
+type UID = string;
```

Comment:
I guess if the type is not `export` then it's groovy, in situ like that? Does that sound right?

Reply:
I think so as it's just alias for documentation and has no particular enforcement or effect on imports.

Can easily be moved later?